prow/cmd/hook/main_test.go

```diff
@@ -28,17 +28,8 @@ import (
 	"k8s.io/test-infra/prow/flagutil"
 	configflagutil "k8s.io/test-infra/prow/flagutil/config"
 	pluginsflagutil "k8s.io/test-infra/prow/flagutil/plugins"
-	"k8s.io/test-infra/prow/plugins"
 )
 
-// Make sure that our plugins are valid.
-func TestPlugins(t *testing.T) {
-	pa := &plugins.ConfigAgent{}
-	if err := pa.Load("../../../config/prow/plugins.yaml", nil, "", true, false); err != nil {
-		t.Fatalf("Could not load plugins: %v.", err)
-	}
-}
```

Comment:
This didn't need to be moved since it doesn't add any validation. `checkconfig` provides all this validation and more.

Reply:
Thanks for the explanation!